packages/prettier-plugin-java/README.md

```diff
@@ -0,0 +1,111 @@
+[![npm](https://img.shields.io/npm/v/prettier-plugin-java.svg)](https://www.npmjs.com/package/prettier-plugin-java)
+
+# prettier-plugin-java
+
+![Prettier Banner](https://raw.githubusercontent.com/prettier/prettier-logo/master/images/prettier-banner-light.png)
+
+Prettier is an opinionated code formatter which forces a certain coding style. It makes the code consistent through an entire project.
+
+This plugin allows the support of Java on Prettier.
+
+The plugin implementation is pretty straightforward as it uses [java-parser](../java-parser) (thanks to Chevrotain) visitor to traverse the **C**oncrete **S**yntax **T**ree and apply the format processing on each node (it uses Prettier API).
+
+## Installation
+
+### Pre-requirements
+
+Since the plugin is meant to be used with Prettier, you need to install it:
+
+`npm install prettier --save-dev`
```

Comment:
it is best to install prettier with exact versions.
I think it is :
`--exact`

See: https://prettier.io/docs/en/install.html